lib/deprecated/pure/parseopt.nim

```diff
@@ -7,6 +7,12 @@
 #    distribution, for details about the copyright.
 #
 
+## **Note**: This module is deprecated since version 0.9.3.
```

Comment:
So both `parseopt` and `parseopt2` are to stay? Shouldn't be there any info what's the purpose of these both?